examples/ReAct/tool_usage.py

```diff
@@ -0,0 +1,71 @@
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+from typing import Dict
+from colorama import Fore
+
+from camel.agents.ReAct_agent import ReActAgent
+from camel.agents.tool_agents.base import BaseToolAgent
+from camel.agents.tool_agents.wiki_tool_agent import WikiToolAgent
+from camel.agents.tool_agents.google_tool_agent import GoogleToolAgent
+
+from camel.generators import SystemMessageGenerator
+from camel.messages import UserChatMessage
+from camel.typing import RoleType, ModelType
+
+
+def main():
```

Comment:
Every example must be tested with pytest, parametrized with at least ModelType.STUB